sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpers.java

```diff
@@ -692,8 +693,9 @@ static String getExtractJobId(String jobIdToken) {
     return String.format("%s-extract", jobIdToken);
   }
 
-  static TableReference createTempTableReference(String projectId, String jobUuid) {
-    String queryTempDatasetId = "temp_dataset_" + jobUuid;
+  static TableReference createTempTableReference(
+      String projectId, String jobUuid, Optional<String> queryTempDatasetIdOpt) {
```

Comment:
Just "tempDatasetId" should be good I think.

Reply:
Changed.